### DISCUSSION TITLE
Is it possible to override an extension and select a different syntax for it?

### DISCUSSION THREAD

**Discussion by eproxus** [sbarex/SourceCodeSyntaxHighlight#227] [Q&A]
I have files with the extension `.config` that I'd like to syntax highlight as _Erlang_ instead of _Configuration file_. Is that possible?

I tried adding a plain file filename pattern with `*.config` and selected Erlang syntax, but it seems the handler for _Configuration file_ takes precedence anyway.

**Comment by JoiHallecksM6** [comment]
Yes, it is possible to configure Visual Studio Code to treat files with a .config extension as Erlang files and provide syntax highlighting accordingly.

**Comment by sbarex** [comment]
You can customize the syntax only for the UTIs (one UTI can be associated to multiple extensions) and not a sigle extension.
You must show the advanced settings (from the view menu), and then the is a `syntax` drop down menu to choose the format from

**Reply by eproxus** [reply]
But that would break any other highlighting for actual _Configuration_ files like `.conf` and `.htaccess`?

**Reply by sbarex** [reply]
unfortunately for the moment yes
